common/display/display-en.html

```diff
@@ -0,0 +1,970 @@
+		This element has an opacity of 1 (100%).
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h3 id="wet-toggling-content">WET - Toggling content</h3>
+
+<h4>WET-BOEW <code>.nojs-show </code>class working example</h4><div class="well">
+	<div class="nojs-show well">
+		This element is initially hidden, but it will be shown when JavaScript is disabled.
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>nojs-show</strong> well"&gt;
+		This element is initially hidden, but it will be shown when JavaScript is disabled.
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>WET-BOEW <code>.nojs-hide </code>class working example</h4><div class="well">
+	<div class="nojs-hide well">
+		This element is initially visible, but it will be hidden when JavaScript is disabled.
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>nojs-hide</strong> well"&gt;
+		This element is initially visible, but it will be hidden when JavaScript is disabled.
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>WET-BOEW <code>.lbx-hide-gal </code>class working example</h4><div class="well">
+	<div class="lbx-hide-gal well">
+		This element is hidden as part of a lightbox gallery.
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>lbx-hide-gal</strong> well"&gt;
+		This element is hidden as part of a lightbox gallery.
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>WET-BOEW <code>.mfp-hide </code>class working example</h4><div class="well">
+	<div class="mfp-hide well">
+		This element is hidden as part of a Magnific Popup modal.
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>mfp-hide</strong> well"&gt;
+		This element is hidden as part of a Magnific Popup modal.
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>WET-BOEW <code>.wb-inv </code>class working example</h4><div class="well">
+	<div class="wb-inv well">
+		This element is invisible and hidden from all users, including screen readers.
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>wb-inv</strong> well"&gt;
+		This element is invisible and hidden from all users, including screen readers.
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>WET-BOEW <code>.wb-invisible </code>class working example</h4><div class="well">
+	<div class="wb-invisible well">
+		This element is invisible but accessible by screen readers.
+	</div>
+</div>
+<h4>Code sample</h4>
+<pre><code>&lt;div class="well"&gt;
+	&lt;div class="<strong>wb-invisible</strong> well"&gt;
+		This element is invisible but accessible by screen readers.
+	&lt;/div&gt;
+&lt;/div&gt;</code></pre>
+
+<h4>WET-BOEW <code>.wb-show-onfocus </code>class working example</h4><div class="well">
+	<input type="text" class="wb-show-onfocus well" placeholder="Focus on this input to reveal the element">
```

Comment:
Avoid placeholder attribute.

This input are missing a label.

Would this example be easier if we try to show a link, like the skip-links?